modin/pandas/accessor.py

```diff
@@ -185,7 +185,7 @@ def to_dense(self):
 
 
 @_inherit_docstrings(pandas.core.accessor.CachedAccessor)
-class CachedAccessor(ClassLogger):
+class CachedAccessor(ClassLogger, modin_layer="ACCESSOR"):
```

Comment:
```suggestion
class CachedAccessor(ClassLogger):
```
This is also pandas API.